text/XXXX-quality-check-workflow.md

```diff
@@ -0,0 +1,546 @@
+| Adapter Content              | [FastQC] | The presence or absence of adapter contamination in your file.          | ![yes](https://img.shields.io/badge/yes-brightgreen) | ![yes](https://img.shields.io/badge/yes-brightgreen) | ![yes](https://img.shields.io/badge/yes-brightgreen) |
+| Status Checks                | [FastQC] | An overview of the pass/warn/fail status for all FastQC checks.         | ![yes](https://img.shields.io/badge/yes-brightgreen) | ![yes](https://img.shields.io/badge/yes-brightgreen) | ![yes](https://img.shields.io/badge/yes-brightgreen) |
+
+[FastQC] rounds out the list of important tools to interrogate the quality of
+the sequencing data itself. We leverage many of the plots produced by FastQC,
+using it as the authority on most sequencing quality metrics, including:
+
+- The **sequencing counts** chart, which we typically ignore in favor of
+  [picard]'s mark duplicates plot.
+- The **sequence quality histogram**, which gives an indication of the
+  sequencing quality across reads. Look for anything that fails FastQC's quite
+  stringent check to see if that correlates with any other failing metrics.
+- The **per sequence quality scores** chart, which shows the mean sequence
+  quality scores across the sample (peaks for lower scores in the orange or red
+  areas of the graph are cause for concern).
+- The **per base sequence content**, which shows is there are any biases in what
+  nucleotides are showing up at particular locations in the reads. For whole
+  genome and whole exome data, there should be relatively few biases. However,
+  for RNA-Seq, you will see a bias at the 5' region of the reads.
+- The **per sequence GC content** chart, which is highly informative of
+  contamination in whole genome and whole exome sequencing data. In those two
+  sequencing modalities, you should check to ensure that all of the
+  distributions are tightly following one another—any derivation from the
+  expected distribution is a sign of contamination. Note that sometimes tools
+  like FastQC plot an "expected" distribution of the GC content; this is usually
+  the expectation of WGS data, but not WXS data, which has a different expected
+  distribution. For RNA-Seq, this graph is generally not used except to examine
+  extreme biases at particular positions.
+- The **per base N content** chart, which should generally report very low
+  levels of Ns in modern data.
+- The **sequence length distribution** chart, which we typically ignore in favor
+  of `ngsderive`'s `readlen` plot.
+- The **sequence duplication level** chart, which gives an indication of the
+  percentage of sequences that are duplicated and by what factor (though this is
+  a graph that we find, often, even good data fails FastQC's stringent test).
+- The **overrepresented sequences** chart, which gives an indication if you're
+  sequencing the same sequence over and over.
+- The **adapter content** chart, which we use to ensure there are minimal to no
```

Comment:
This is a good example of required background knowledge. By contrast, above you explain what `C` and `G` stand for and describe the nucleotide bonding. Here we assume that the reader is familiar with sequencing adapters.